src/flexbox/nasa-style.css

```diff
@@ -0,0 +1,73 @@
+* {
+  box-sizing: border-box;
+}
+
+body{
+  margin:0;
+  padding:0;
+}
+
+.main-container {
+  height: 100%;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.header-container {
+  background-color: lightblue;
+  width: 100%;
+  height: 450px;
+  background-image: url(./img/nasa-background.jpg);
+  background-repeat: no-repeat;
+  background-size: 200%;
+  background-position: center;
+  display: flex;
+  color: white;
+  justify-content: center;
+  align-items: center;
+  overflow: hidden;
+}
+
+.header-container h1 {
+  font-size: 5em;
+  z-index: 2;
+}
+
+.info-container {
+  background-color: rgb(230, 230,230);
+  width: 100%;
+  height: 350px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+  margin-top: 20px;
+}
+
+.info-card {
+  /* border: 1px red solid; */
+  width: 30%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  padding-left: 20px;
+  overflow: hidden;
+}
+
+@media (max-width: 380px) {
```

Comment:
change this to 576px and adjust. definitely right about `flex-direction: column;`